internal/cmd/base/base.go

```diff
@@ -116,7 +116,7 @@ type Command struct {
 	FlagSkipCacheDaemon  bool
 	FlagSkipClientAgent  bool
 
-	FlagClientAgentPort uint
+	FlagClientAgentPort uint64
```

Comment:
Can we make this a uint16? Ports on systems are limited to this number already.

Reply:
+1

Reply:
The only blocker to changing this to `uint16` is that we then couldn't pass a pointer to this variable to all the `f.Uint64Var` calls we do.

Reply:
Yeah, Louis and I were looking at the possibility of adding that type in, should be simple 👍

Reply:
All done - See [df25e75](https://github.com/hashicorp/boundary/pull/5027/commits/df25e75990daedd9b3b6620f6bd9eb8e8855d011) and https://github.com/hashicorp/boundary/pull/5027/commits/b0880a8fd941ca3db45cad5273d15c88bc0730eb